food/activities.json

```diff
@@ -1325,7 +1325,7 @@
     "name": "Truite d'élevage FR Conv.",
     "process_categories": ["ingredient"],
     "raw_to_cooked_ratio": 0.819,
-    "search": "Large trout, 2-4kg, conventional, at farm gate/FR U :",
+    "search": "Large trout, 2-4kg, conventional, at farm gate FR U",
```

Comment:
[`/slap`](https://en.wiktionary.org/wiki/trout_slap)

(sorry that's pretty much all I can bring to the table as a reviewer right now)

Reply:
Not sure if the fact that I understand the joke is a good news for me 😭😅